### ISSUE TITLE
Add wait time to loans page

### ISSUE THREAD

**Issue by BrittanyBunk** [internetarchive/openlibrary#2938]
https://openlibrary.org/account/loans
![image](https://user-images.githubusercontent.com/55302685/73473420-9a5ddf80-4341-11ea-81ce-9d34319aa369.png)
This helps a person know how long they'll wait until a book is available, which is really valuable to know. Without this knowledge - people might unfortunately miss their waitlist spot - which means they may wait for nothing - which isn't good. Or, because of not knowing when they'll be next on the waitlist, they don't check frequently enough that they hold others up - by staying on the waitlist when it becomes available instead of checking out the book - which holds up the line for others. My solution prevents all that.

**Comment by tfmorris** [comment]
I though users got email when the book was available for them.

What would the estimate be based on ?

**Comment by BrittanyBunk** [comment]
@tfmorris right. But the issue is that to know when the email pops up is uncertain without knowing when the wait time is up. If a person's not paying attention to their email at the right time, they might not get to it and then (idk exactly how this works) it could go to the next person.

It's based on how long people get books + where the person with the book has it. Example: 3 people on the waitlist (including me if I am on the waitlist). The person with the book is 4 days in to the book (has 10 days left). So that's 4 people total. Each person on the waitlist gets 14 days with the book + however long they can be on the waitlist for before they start the 14 days (I'm just going to say 1 week, because idk exactly). So the estimated wait time is 10 days (for the 1st person) + 3 (the 3 waitlist people) x (7 days (by having it in the email) + 14 days (with the book)) = 72 days or 10 weeks and 2 days.
